### PR TITLE
Hero Balance Suggestions

### PR DESCRIPTION
**Pull request by imaGecko**
Treant
Leech Seed damage/heal increased from 15/30/45/60/90/210 to 15/30/45/60/120/270
Living Armor Regeneration increased from 4/8/12/16/24/56 to 4/8/12/16/32/96
Living Armor Damage Block increased from 20/40/60/80/120/280 to 20/40/60/80/160/380
Living Armor Duration reduced from 15/15/15/15/16/18 to 15 at all levels

Slight buff to his damage, overall buffed healing. Should make him much more useful as a support.
Removed the useless entries in the eyes of the forest kv as it is an ability that cannot be leveled.

Timbersaw
Reactive Armor armor/regen per stack increased from 1/1.2/1.4/1.6/1.8/2.2 to 1/1.2/1.4/1.6/2.0/2.6
Maximum stacks reduced from 20/20/20/20/25/35 to 20 at all levels

This will cause him to be tankier at less stacks but less tanky at full stacks. 35 stacks take too long to achieve to achieve and are unreasonably hard to hold. Timbersaw already has the damage he needs, this should be enough to make him competitive again.

Outworld Devourer
Changing the KV from pure to physical has no effect on the ability, it only changes the tooltip. Reverting the change as right now it is misleading and false.

### CODE REVIEW

**Comment by imaGecko** [comment]
would need someone to review this

**Comment by Haganeko** [comment]
Unsure about the Timber fix - later feedback indicates he's tanky enough as is - but I guess we can reverse it as necessary. Also, did the OD physical work?

**Comment by imaGecko** [comment]
Looks like the OD change also did nothing besides changing the tooltip

**Comment by Haganeko** [comment]
WIP label until it can get wider review.

**Comment by Haganeko** [comment]
WIP label until it can get wider review.

**Comment by Haganeko** [comment]
Should remove the silencer change now that the ability has been rewritten.

**Comment by Nexren** [comment]
Bring back the dmage for Doom's Scorching earth and Ulti

File: game/scripts/npc/abilities/shredder_reactive_armor.txt [hunk 1]
```diff
@@ -23,12 +23,12 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "bonus_armor"                                     "1 1.2 1.4 1.6 2.25 3.85"
+        "bonus_armor"                                     "1 1.2 1.4 1.6 2.0 2.6"
```
Comment (Haganeko) [inline game/scripts/npc/abilities/shredder_reactive_armor.txt]: Is there a need for this? I've seen no discussion of it.

**Comment by imaGecko** [comment]
Doom's damage and various others should be fixed with #1255

File: game/scripts/npc/abilities/shredder_reactive_armor.txt (hunk 1 above)
Comment (imaGecko) [inline game/scripts/npc/abilities/shredder_reactive_armor.txt]: Did I completely misremember? My mind has been a little bit scrambled lately

**Comment by Haganeko** [comment]
Should we close this, or at least look at it again? It's been sitting here for  a while, and I like the changes (except maybe for OD, if we implemented the change).

**Comment by imaGecko** [comment]
Could also remove some things to at least get the treant buff out

**Comment by Haganeko** [comment]
I approve of this idea.

**Pull request closed without merging**